src/backend/app/Http/Controllers/RevenueController.php

```diff
@@ -152,7 +152,7 @@ public function revenueData(Request $request): ApiResource {
                     'average_revenue_per_month' => '-',
                 ];
             }
-        } elseif ($targets !== null && $targetType === 'mini-grid' && isset($targets->subTargets)) {
+        } elseif ($targetType === 'mini-grid' && isset($targets->subTargets)) {
```

Comment:
I'm concerned about removing the null check, is it guaranteed not never ever be null?

Reply:
I missed that 😄. Thanks. I can be less woried now